.github/workflows/checkbox-promote-beta-to-candidate/job.template

```diff
@@ -35,24 +35,23 @@ test_data:
     # install checkbox runtime on device (snap or deb)
     if [[ "$SNAP_FLAG" == "true" ]]; then
         echo "Installing checkbox runtime on device (from snap)"
-        CHANGE_ID=$(_run sudo snap install $CHECKBOX_RUNTIME --no-wait --channel=latest/$CHANNEL)
-        wait_for_snap_change_id $CHANGE_ID
+        _run_retry sudo snap install $CHECKBOX_RUNTIME --channel=latest/$CHANNEL
         _run_retry sudo snap install checkbox --devmode --channel=$CHECKBOX_TRACK/$CHANNEL
         # list all the installed snaps to facilitate debugging
         _run snap list
         CHECKBOX_CLI_CMD="checkbox.checkbox-cli"
     else
         echo "Installing checkbox runtime on device (from deb package)"
         _run sudo add-apt-repository -y ppa:checkbox-dev/$CHANNEL
-        _run install_packages checkbox-*
+        _run sudo DEBIAN_FRONTEND=noninteractive apt-get -qq install -o Dpkg::Options::="--force-confdef" -y --force-yes checkbox-ng python3-checkbox-ng checkbox-provider-base checkbox-provider-resource checkbox-provider-sru fswebcam obexftp wmctrl iperf mesa-utils vim pastebinit fwts xorg-dev gir1.2-clutter-1.0
```

Comment:
Why not `install_packages checkbox-ng python3-checkbox-ng checkbox-provider-base checkbox-provider-resource checkbox-provider-sru fswebcam obexftp wmctrl iperf mesa-utils vim pastebinit fwts xorg-dev gir1.2-clutter-1.0`?

Reply:
Sloppy copy paste from the sru template, updated